### DISCUSSION TITLE
Applications keep getting rejected

### DISCUSSION THREAD

**Discussion by mustiej** [community/community#58753] [GitHub Education]
### Hi everyone,

I've done 4 applications to benefit from the Students Pack but unfortunately it's been rejected by the following message:

- You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

I've tried to resolve this but no luck.

**Comment by the-officialjosh** [comment]
Please proceed with making your profile public and ensure that the order of the information in your profile aligns with the document, such as 'sur - first order' or vice versa. and Bio too

**Reply by mustiej** [reply]
I tried with public profile before with filled in bio and correct order of full name. It still got rejected.

**Reply by the-officialjosh** [reply]
what document are you using. My younger bro used his ID card and it got approved in just 2 days

**Reply by mustiej** [reply]
I used proof of enrollment, it worked twice before.

**Reply by the-officialjosh** [reply]
Honestly, it seems that I have reached the limit of my knowledge regarding this particular issue.
